bundles/org.palladiosimulator.dataflow.confidentiality.analysis/src/org/palladiosimulator/dataflow/confidentiality/analysis/dsl/AbstractStandalonePCMDataFlowConfidentialityAnalysis.java

```diff
@@ -0,0 +1,229 @@
+package org.palladiosimulator.dataflow.confidentiality.analysis.dsl;
```

Comment:
I'm not sure about the `dsl` in the package name. I would not see this as a dsl but rather as the analysis core classes

Reply:
Besides this, I like the separation in abstract, legacy and normal